### ISSUE TITLE
Update a `file_from_gdrive` model

### ISSUE THREAD

**Issue by orsk-moscow** [orsk-moscow/utilities#44]
### Description

```markdown
**Goal**
Update a `file_from_gdrive` model.
```

### Additional context

```plain text
**Details**
A model for the file_from_gdrive (link 
https://github.com/orsk-moscow/utilities/blob/72981f04a00963ff4413e349f5d7edfb6501d9c8/models.py#L11) should be updated since another field could be used as a `datetime` attribute: 

greater_field: Literal["createdTime", "modifiedTime"] = "createdTime",
```

That is an enhancement, not a bug or a feature.
This need to be done for clarification.
```


### Urgency

Issue can be postponed

### Priority Prod Data.

No

### Priority of a Time.

No

### Priority of a Money 1.

No

### Priority of a Money 2.

No

### Blocks

-

### Blocked by

-

### Size of related Repositories

utilities

### Size of Pipelines

_No response_

### Size of Data

Existing data source

### Size of documentation

Existing documentation, New documentation

### Size of Test coverage

Unit tests already cover the issue, Unit tests need to be updated, Unit tests need to be written from scratch

### Acceptance criteria

```markdown
-
```

### Specific test cases

```markdown
-
```
